pkgs/development/python-modules/rendez/default.nix

```
@@ -0,0 +1,59 @@
+{
+  asgiref,
+  buildPythonPackage,
+  click,
+  cryptography,
+  fetchgit,
+  flask,
+  flit-core,
+  highctidh,
+  ifaddr,
+  lib,
+  pymonocypher,
+  pysocks,
+  requests,
+  stem,
+  toml,
+  unittestCheckHook,
+}:
+buildPythonPackage {
+  pname = "rendez";
+  version = "unstable-2025-04-20";
```

https://github.com/NixOS/nixpkgs/blob/ea517c04f2fd39382057616b0cadfe979b70626f/pkgs/README.md#versioning

```suggestion
  version = "1.2.1-unstable-2025-04-20";
```

And maybe consider adding a `passthru.updateScript` with `unstableGitUpdater`, to make updating the package and keeping this value correct an automated process :slightly_smiling_face:.

---

Thank you @OPNA2608! What if I expect upstream to make releases?

---

The latter, please.